packages/client/test/miner/miner.spec.ts

```diff
@@ -368,6 +369,79 @@ tape('[Miner]', async (t) => {
     txPool.stop()
   })
 
+  t.test('should handle mining over the london hardfork block', async (t) => {
+    const customChainParams = {
+      hardforks: [
+        { name: 'chainstart', block: 0 },
+        { name: 'berlin', block: 2 },
+        { name: 'london', block: 3 },
+      ],
```

Comment:
Ah, funny, I guess that's the first time (that I see) that we are totally detaching from the history-given HF sequence. 😀